.github/workflows/edge.yml

```
@@ -0,0 +1,69 @@
+---
+name: Edge
+
+# The "Edge Test" is a run of the agent test suite with pre-release builds
+# of node.js, if available and useful. "Pre-release" builds are release
+# candidate (RC) and "nightly" node.js builds.
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '**.md'
+      - '**.asciidoc'
```

```suggestion
      - '**.asciidoc'
      - 'docs/**'
      - 'examples/**'
```

Could perhaps add these as we have in "test.yml".